server/api/shares.js

```diff
@@ -87,22 +123,25 @@ router.post("shares.create", auth(), async ctx => {
     },
   });
 
-  await Event.create({
-    name: "shares.create",
-    documentId,
-    collectionId: document.collectionId,
-    modelId: share.id,
-    teamId: user.teamId,
-    actorId: user.id,
-    data: { name: document.title },
-    ip: ctx.request.ip,
-  });
+  if (isCreated) {
```

Comment:
fixes: `shares.create` events were being created whenever the share link dialog is opened.